### PR TITLE
Cybrics 2020 writeups

### PR DESCRIPTION
**Pull request by Pharisaeus**

### CODE REVIEW

File: 2020-07-25-CybricsCTF/xcorp/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# XCorp (network, 50p, 159 solved)
+
+In the task we get a [pcap](xcorp.pcap) to analyse.
+Networ Miner recovers [executable](main.exe) for us included in the pcap.
```
Comment (XeR) [inline 2020-07-25-CybricsCTF/xcorp/README.md:4]: There is no main.exe ; did you mean net10.exe ?

Comment (Pharisaeus) [reply 2020-07-25-CybricsCTF/xcorp/README.md:4]: Yes, that's the one :)

**Comment by XeR** [comment]
Last commit uses your work identity (name + e-mail).
Should we change it ?

**Comment by Pharisaeus** [comment]
Wasted effort since I have non-existent opsec anyway ;)

**Pull request merged**